llama_stack/providers/utils/inference/model_registry.py

```diff
@@ -57,13 +57,13 @@ def __init__(self, model_aliases: List[ModelAlias]):
             self.alias_to_provider_id_map[alias_obj.llama_model] = alias_obj.provider_model_id
             self.provider_id_to_llama_model_map[alias_obj.provider_model_id] = alias_obj.llama_model
 
-    def get_provider_model_id(self, identifier: str) -> str:
+    def get_provider_model_id(self, identifier: str) -> Optional[str]:
         if identifier in self.alias_to_provider_id_map:
```

Comment:
whole thing could be just `self.alias_to_provider_id_map.get(identifier, None)` I think.